dracut/30ignition/module-setup.sh

```diff
@@ -57,6 +69,17 @@ install() {
     inst_simple "$moddir/ignition" \
         "/usr/bin/ignition"
 
+    # Rule to allow udev to discover unformatted encrypted devices
+    inst_simple "$moddir/99-xx-coreos-systemd-cryptsetup.rules" \
+        "/usr/lib/udev/rules.d/99-xx-coreos-systemd-cryptsetup.rules"
```

Comment:
nit: s/coreos/ignition/ ?

Why `-xx-`?

Reply:
That rule was plucked from @lucab's [old CL PR](https://github.com/coreos/bootengine/pull/141/files) so I left it with the naming from there. Can rename & drop the `-xx-`.

Reply:
Actually looking at it again we're overriding `99-systemd.rules` with this so we need the `-xx-`. Will change it to `99-xx-ignition-systemd-cryptsetup.rules`